src/java/apiview-java-processor/src/main/java/com/azure/tools/apiview/processor/analysers/ASTAnalyser.java

```diff
@@ -237,6 +240,119 @@ private void visitClassOrInterfaceOrEnumDeclaration(TypeDeclaration<?> typeDecla
             addToken(new Token(NEW_LINE, ""));
         }
 
+        private void visitModuleDeclaration(ModuleDeclaration moduleDeclaration) {
+            addToken(new Token(KEYWORD, "module"));
+            addToken(new Token(WHITESPACE, " "));
+            addToken(new Token(TYPE_NAME, "module-info", "module-info"));
```

Comment:
does the definitionId="module-info" change to a specific id?

Reply:
no, because it is a special case it can just be hard-coded.